src/AnonymousFactory.php

```diff
@@ -11,29 +11,33 @@
 
 namespace Zenstruck\Foundry;
 
+use Zenstruck\Foundry\Persistence\PersistentObjectFactory;
+
 /**
  * @template TModel of object
- * @template-extends Factory<TModel>
  *
  * @author Kevin Bond <kevinbond@gmail.com>
  *
  * @deprecated
  */
-final class AnonymousFactory extends Factory implements \Countable, \IteratorAggregate
+final class AnonymousFactory implements \Countable, \IteratorAggregate
```

Comment:
we cannot extend `Factory` anymore because of the method static `new` which now exists in `BaseFactory` and does not have the same prototype